packages/jmix-react-antd/src/ui/form/controls/SliderField.tsx

```diff
@@ -0,0 +1,32 @@
+import { Slider } from "antd";
+import * as React from "react";
+import { JmixFormFieldWrapper } from "./base/JmixFormFieldWrapper";
+import { SliderSingleProps, SliderRangeProps } from "antd/es/slider";
+import { JmixFormFieldProps } from "./base/JmixFormFieldProps";
+
+type SliderProps = (SliderSingleProps | SliderRangeProps) & JmixFormFieldProps;
```

Comment:
Add export for this type and rename to SliderFieldProps please.
```suggestion
export type SliderFieldProps = (SliderSingleProps | SliderRangeProps) & JmixFormFieldProps;
```
After that ready to merge